### PR TITLE
Add Hibernate Panache in Kotlin Quickstart

### PR DESCRIPTION
**Pull request by sixdouglas**
**Check list**:

Your pull request:

- [X] targets the `development` branch
- [X] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] has native tests (`mvn clean verify -Pnative`)
- [x] makes sure the associated guide must not be updated
- [x] links the guide update pull request (if needed)
- [x] updates or creates the `README.md` file (with build and run instructions)
- [X] for new quickstart, is located in the directory _component-quickstart_
- [X] for new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

File: hibernate-orm-panache-kotlin-quickstart/pom.xml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.acme</groupId>
+    <artifactId>hibernate-orm-panache-kotlin-quickstart</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <properties>
+        <quarkus-plugin.version>2.3.0.Final</quarkus-plugin.version>
```
Comment (loicmathieu) [inline hibernate-orm-panache-kotlin-quickstart/pom.xml:11]: should be 999-SNAPSHOT

File: hibernate-orm-panache-kotlin-quickstart/pom.xml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+        <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+        <quarkus.platform.version>2.3.0.Final</quarkus.platform.version>
```
Comment (loicmathieu) [inline hibernate-orm-panache-kotlin-quickstart/pom.xml:14]: Should be 999-SNAPSHOT

**Review by loicmathieu — APPROVED** [review]
LGTM, thanks @douglas-six.

**Comment by glefloch** [comment]
@loicmathieu can this be merge ? it may need a rebase.

**Comment by gsmet** [comment]
I rebased it and updated a few things (Maven wrapper and Kotlin version).

**Comment by loicmathieu** [comment]
@glefloch @gsmet thanks for taking care of it, I forgot this PR.

@sixdouglas you should have ask me about it ! :)

**Comment by gsmet** [comment]
Thanks @sixdouglas

**Pull request merged**